markdown/src/viewer.c

```diff
@@ -300,7 +300,7 @@ static void
 on_webview_load_status_notify(WebKitWebView *view, GParamSpec *pspec,
   MarkdownViewer *self)
 {
-  WebKitLoadStatus load_status;
+  WebKitLoadEvent load_status;
 
   g_object_get(view, "load-status", &load_status, NULL);
```

Comment:
[webkit2gtk web view](https://webkitgtk.org/reference/webkit2gtk/stable/WebKitWebView.html) doesn't ave a `load-status` property.

this should be replaced with [`load-changed`](https://webkitgtk.org/reference/webkit2gtk/stable/WebKitWebView.html#WebKitWebView-load-changed) and/or [`load-failed`](https://webkitgtk.org/reference/webkit2gtk/stable/WebKitWebView.html#WebKitWebView-load-failed) handlers.

Reply:
There's an implementation of that here: 
https://github.com/hyperair/geany-plugins/commit/1fd266a97b9fedb0ae463a2cf9a409af27bb0ae6?diff=unified#diff-6849d7a7a6ff4110bd13706035ccd7daL545
https://github.com/hyperair/geany-plugins/commit/1fd266a97b9fedb0ae463a2cf9a409af27bb0ae6?diff=unified#diff-6849d7a7a6ff4110bd13706035ccd7daR1195